### DISCUSSION TITLE
changing the base template for one particular Laravel Backpack CRUD

### DISCUSSION THREAD

**Discussion by prescriptionlifeline** [Laravel-Backpack/community-forum#981] [Q&A (Help)]
I'd like to make it so that a particular Laravel Backpack CRUD uses resources/views/layouts/app.blade.php vs whatever it's using right now (backpack/theme-tabler/inc/head.blade.php?)

I mean, I'd still like to use all the CRUD Field template components but I want to use the same template that my non Laravel Backpack pages use.

Any ideas as to how I might do this?

**Comment by prescriptionlifeline** [comment]
In the `setup()` method I added this:

```php
CRUD::setListView('customView');
```

In customView.blade.php I have this:

```
<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <meta name="csrf-token" content="{{ csrf_token() }}">

        <title>{{ config('app.name', 'Laravel') }}</title>

        <!-- Fonts -->
        <link rel="preconnect" href="https://fonts.bunny.net">
        <link href="https://fonts.bunny.net/css?family=figtree:400,500,600&display=swap" rel="stylesheet" />
        <link rel="icon" href="/logo.svg">

        <!-- Scripts -->
        @vite(['resources/css/app.css', 'resources/js/app.js'])
    </head>
    <body class="font-sans antialiased">
        <div class="min-h-screen bg-gray-100">
            @include('layouts.navigation')

            <!-- Page Heading -->
            @if (isset($header))
                <header class="bg-white shadow">
                    <div class="max-w-7xl mx-auto py-6 px-4 sm:px-6 lg:px-8">
                        {{ $header }}
                    </div>
                </header>
            @endif

            <!-- Page Content -->
            <main>
WHAT GOES HERE?
            </main>
        </div>
    </body>
</html>
```
So what goes in the `WHAT GOES HERE?` section?

I tried `@yield('content')` but that didn't do anything. I also tried to copy the stuff in `@section('content')` from vendor/backpack/crud/src/resources/views/crud/list.blade.php into that area but no data is actually showing up.

If I copy the contents of vendor/backpack/crud/src/resources/views/crud/list.blade.php to customView.blade.php it shows up as it would if I wasn't using a custom template at all.

**Comment by karandatwani92** [comment]
Hey @prescriptionlifeline 

Are you thinking of replacing the head section so you can change the theme? NO, Replacing the head is complicated.

Backpack has multiple theme support, and other assets are dynamically included as per field & column usage. This all makes the head section way dynamic.
 
I would recommend you do the following instead:
1. [Create your custom backpack theme](https://backpackforlaravel.com/docs/6.x/add-ons-tutorial-how-to-create-a-theme).
2. Then dynamically change it for a particular CRUD or Operation via:
```
public function setup()
{
   ...
   Config::set('backpack.ui.view_namespace', 'backpack.theme-coreuiv4::');
}
```
This way, you'll also be free to use your theme for the whole admin panel to match it with the front end.

And, If you would ❤️  to share your theme with other backpack users, you can [create a theme as a package](https://backpackforlaravel.com/docs/6.x/add-ons-tutorial-how-to-create-a-theme#part-b-create-the-package).
